app/views/posts/show.html.erb

```diff
@@ -32,11 +32,20 @@
         <p><%= comment.author.name %>: </p>
         <p><%= comment.text %></p>
       </div>
+      <% if can? :destroy, comment %>
+      <%= form_with(model: comment, url: comment_path(comment), method: :delete, data: { turbo: "false" }) do |form| %>
+        <%= form.submit "Delete this comment", class: "delete-post top-right-cornor" %>
+      <% end %>
```

Comment:
- [ ] Kindly enable post deletion, the admin or the owner of a post or comment should be able to delete them, but currently, only a comment is deleted.

![delete](https://github.com/kalbek/blog-app/assets/53124548/66e19528-85bb-4d52-95a3-4b182d22e0a0)